qemu/tests/cfg/virtio_fs_share_data_multi_backend.cfg

```diff
@@ -69,9 +69,24 @@
                     export_dir_fs2 = /mnt/virtio_fs2_test_nfs
                     nfs_mount_src_fs1 = 127.0.0.1:/mnt/virtio_fs1_test_nfs
                     nfs_mount_src_fs2 = 127.0.0.1:/mnt/virtio_fs2_test_nfs
+        - with_ext_fs:
+            setup_filesystem_on_host = yes
+            dd_of_on_host = '/home/extdisk_virtiofs_test'
+            cmd_losetup_query_on_host = 'losetup -f'
+            cmd_losetup_on_host = 'losetup -f %s'
+            cmd_losetup_d_on_host = 'losetup -d %s'
+            cmd_mkfs_on_host = 'mkfs.%s '
+            cmd_dd_on_host = 'dd if=/dev/urandom of=%s bs=1M count=1024 oflag=direct'
```

Comment:
Hit error on arm64:
output: "dd: error writing '/mnt/myfs/test_file': No space left on device\n9
74+0 records in\n973+0 records out\n1020264448 bytes

Check on arm guest:

```
[root@dhcp19-243-209 ~]# df -h
myfs                                   975M   60K  924M   1% /mnt/myfs
```

Since the size of dd in the test is 1G, so recommended to create a size greater than 1G here. e.g. 1.2G

Reply:
> Hit error on arm64: output: "dd: error writing '/mnt/myfs/test_file': No space left on device\n9 74+0 records in\n973+0 records out\n1020264448 bytes
> 
> Check on arm guest:
> 
> ```
> [root@dhcp19-243-209 ~]# df -h
> myfs                                   975M   60K  924M   1% /mnt/myfs
> ```
> 
> Since the size of dd in the test is 1G, so recommended to create a size greater than 1G here. e.g. 1.2G

Hello @zhenyzha , the size has been increased to 2G. Please check again, thanks!

Reply:
sorry the size of dd in the test is 2G, so recommended to create a size greater than 2G here. e.g. 2660

https://github.com/autotest/tp-qemu/blob/master/qemu/tests/cfg/virtio_fs_share_data_multi_backend.cfg#L114

'dd if=/dev/urandom of=/mnt/myfs/test_file bs=1M count=2048 of
lag=direct'    (status: 1,    output: "dd: error writing '/mnt/myfs/test_file': No space left on device\n1947+0 records in\n1946+0 records out\n2040528896 bytes (2.0 GB,

Reply:
> sorry the size of dd in the test is 2G, so recommended to create a size greater than 2G here. e.g. 2660
> 
> https://github.com/autotest/tp-qemu/blob/master/qemu/tests/cfg/virtio_fs_share_data_multi_backend.cfg#L114
> 
> 'dd if=/dev/urandom of=/mnt/myfs/test_file bs=1M count=2048 of lag=direct' (status: 1, output: "dd: error writing '/mnt/myfs/test_file': No space left on device\n1947+0 records in\n1946+0 records out\n2040528896 bytes (2.0 GB,

The size is 2660MB. Please take a look, thanks!